hotsos/core/host_helpers/systemd.py

```diff
@@ -86,25 +86,36 @@ def start_time_secs(self):
         return t.timestamp()
 
     @property
-    def memory_current(self):
-        """ Returns service current memory usage in bytes. """
-        path = os.path.join(HotSOSConfig.data_root,
-                            'sys/fs/cgroup/memory/system.slice',
-                            "{}.service".format(self.name),
-                            'memory.usage_in_bytes')
+    def memory_current_kb(self):
+        """ Returns service current memory usage in kbytes.
+
+        See https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt
+        """
+        cgroup_base = os.path.join(HotSOSConfig.data_root, 'sys/fs/cgroup')
+        path = os.path.join(cgroup_base,
+                            "memory/system.slice/{}.service".format(self.name),
+                            'memory.stat')
         if not os.path.exists(path):
             # path changed between Ubuntu Focal and Jammy releases.
-            path = os.path.join(HotSOSConfig.data_root,
-                                'sys/fs/cgroup/system.slice',
+            path = os.path.join(cgroup_base, 'system.slice',
                                 "{}.service".format(self.name),
-                                'memory.current')
+                                'memory.stat')
 
-        if not os.path.exists(path):
-            log.warning("service memory info not found at %s", path)
-            return 0
+        total_usage = {}
+        fs = FileSearcher()
+        fs.add(SearchDef(r'(cache|rss|swap) (\d+)'), path=path)
```

Comment:
memory.stat does not contain cache/rss/swap values in [cgroup v2](https://www.kernel.org/doc/Documentation/cgroup-v2.txt) (jammy and above).

Reply:
going to stick with memory.current for cgroup-v2 unless we think that is also not reliable